src/model.jl

```diff
@@ -134,61 +133,61 @@ function EMB.variables_capex(m, 𝒩, 𝒯, 𝒫, modeltype::AbstractInvestmentM
 end
 
 """
-    EMB.constraints_capacity_installed(m, n::EMB.Node, 𝒯, modeltype::AbstractInvestmentModel
+    EMB.constraints_capacity_installed(
+        m,
+        n::EMB.Node,
+        𝒯::TimeStructure,
+        modeltype::AbstractInvestmentModel,
+    )
 
-Set capacity-related constraints for nodes `𝒩` for investment time structure `𝒯`:
-* bounds
-* binary for BinaryInvestment
-* link capacity variables
+When the modeltype is an investment model, the function introduces the related constraints
```

Comment:
```suggestion
Add capacity expansion constraints to model `m` when `modeltype` is investment model. 
```

Reply:
I am a bit opposed to the rephrasing here. The approach of using `when` highlights with change has to happen so that we use the function.